projects/astherus/index.js

```diff
@@ -16,14 +18,18 @@ Object.keys(config).forEach(chain => {
     tvl: async (api) => {
       const { data } = await getConfig(`astherus/${api.chain}`, `https://astherus.finance/bapi/futures/v1/public/future/web3/ae-deposit-asset?chainId=${api.chainId}`)
       const tokens = data.map(i => i.contractAddress)
+      if (chain === 'bsc') {
+        const asBTCTVL = await api.call({ abi: 'erc20:totalSupply', target: asBTCContract })
+        api.addToken(asBTCContract, asBTCTVL)
```

Comment:
thanks for the PR, but dont think this should be part of derivatives product, also, is there some way for us to pull tvl data on chain? like ceffu has dedicated wallet for your BTC where we can look up balance instead of taking total supply?